src/Sarif.UnitTests/SarifVersionOneToCurrentTests.cs

```diff
@@ -1453,5 +1483,336 @@ public void SarifTransformerTests_ToCurrent_TwoResultsWithFixes()
 
             v2LogText.Should().Be(v2LogExpectedText);
         }
+
+        [Fact]
+        public void SarifTransformerTests_ToCurrent_CodeFlows()
+        {
+            string v1LogText =
+@"{
+  ""version"": ""1.0.0"",
+  ""runs"": [
+    {
+      ""tool"": {
+        ""name"": ""CodeScanner"",
+        ""semanticVersion"": ""2.1.0""
+      },
+      ""logicalLocations"": {
+        ""collections::list::add"": {
+          ""name"": ""add"",
+          ""kind"": ""function"",
+          ""parentKey"": ""collections::list""
+        },
+        ""collections::list"": {
+          ""name"": ""list"",
+          ""kind"": ""type"",
+          ""parentKey"": ""collections""
+        },
+        ""collections"": {
+          ""name"": ""collections"",
+          ""kind"": ""namespace""
+        }
+      },
+      ""results"": [
+        {
+          ""ruleId"": ""C2001"",
+          ""formattedRuleMessage"": {
```

Comment:
We don't need rule metadata to test code flows. Just use `result.message`.

Reply:
OK